_posts/2016-05-04-announcing-scholarship-applications.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+excerpt_separator: <!--more-->
+title: Announcing scholarship applications 2016
+---
+
+<h3>Mission</h3>
+
+<p>
+  GORUCO’s diversity scholarship program aims to support
+  traditionally underrepresented and/or marginalized persons
+  in the Ruby community (including, but not limited to
+  persons identifying as women, persons of colour, LGBT, and/or
+  persons with disabilities) who may not otherwise have the opportunity
+  to attend the conference for financial reasons.</p>
```

Comment:
I would probably change "persons" to "people" and "colour" to "color" to Americanize it.